### ISSUE TITLE
already started error_logger

### ISSUE THREAD

**Issue by sebastiw** [cvik/leg#2]
leg crashes with badmatch.
```
=INFO REPORT==== 24-Jul-2018::11:55:44 ===
    application: leg
    exited: {bad_return,
                {{leg_app,start,[normal,no_arg]},
                 {'EXIT',
                     {{badmatch,{error,{already_started,<0.779.0>}}},
                      [{leg_error_logger_handler,clear_error_logger,0,
                           [{file,
                                "/artifact/_build/default/lib/leg/src/leg_error_logger_handler.erl"},
                            {line,76}]},
```
It's on this line `ok = error_logger:start(),`, I think `{error, {already_started, _}}` should be allowed here as well. I will add a PR when I have some time.

**Comment by sebastiw** [comment]
#3

**Comment by cvik** [comment]
Solved by PR #3.
